learn/cookbooks/langchain.mdx

```diff
@@ -0,0 +1,187 @@
+
+## Performing similarity search
+
+Your database should now contain the movies. Create a new `search.py` file to make a semantic search query: searching for documents using similarity search.
+
+```python
+# search.py
+
+import os
+from dotenv import load_dotenv
+from langchain.vectorstores import Meilisearch
+from langchain.embeddings.openai import OpenAIEmbeddings
+import meilisearch
+
+load_dotenv()
+
+# You can use the same code as `setup.py` to check for missing env vars
+
+# Create the vector store
+client = meilisearch.Client(
+    url=os.environ.get("MEILI_HTTP_ADDR"),
+    api_key=os.environ.get("MEILI_API_KEY"),
+)
+embeddings = OpenAIEmbeddings()
+vector_store = Meilisearch(client=client, embedding=embeddings)
+
+# Make similarity search
+embedder_name = "custom"
+query = "superhero fighting evil in a city at night"
+results = vector_store.similarity_search(
+    query=query,
+    embedder_name=embedder_name,
+    k=3,
+)
+
+# Display results
+for result in results:
+    print(result.page_content)
+```
+
+Run our file. If everything works, You should see an output like this:
+
+```
+{"id": 155, "title": "The Dark Knight", "overview": "Batman raises the stakes in his war on crime. With the help of Lt. Jim Gordon and District Attorney Harvey Dent, Batman sets out to dismantle the remaining criminal organizations that plague the streets. The partnership proves to be effective, but they soon find themselves prey to a reign of chaos unleashed by a rising criminal mastermind known to the terrified citizens of Gotham as the Joker."}
+{"id": 314, "title": "Catwoman", "overview": "Liquidated after discovering a corporate conspiracy, mild-mannered graphic artist Patience Phillips washes up on an island, where she's resurrected and endowed with the prowess of a cat -- and she's eager to use her new skills ... as a vigilante. Before you can say \"cat and mouse,\" handsome gumshoe Tom Lone is on her tail."}
+{"id": 268, "title": "Batman", "overview": "Batman must face his most ruthless nemesis when a deformed madman calling himself \"The Joker\" seizes control of Gotham's criminal underworld."}
+```
+
+Congrats 🎉 You managed to make a similarity search using Meilisearch as a LangChain vector store.
+
+## Going further
+
+Using Meilisearch as a LangChain vector store allows you to load documents and search for them in different ways:
+
+- [Import documents from text](https://python.langchain.com/docs/integrations/vectorstores/meilisearch#adding-text-and-embeddings)
+- [Similarity search with score](https://python.langchain.com/docs/integrations/vectorstores/meilisearch#similarity-search-with-score)
+- [Similarity search by vector](https://python.langchain.com/docs/integrations/vectorstores/meilisearch#similarity-search-by-vector)
+
+For additional information, make sure to consult:
```

Comment:
For additional information on what? Also, I'd suggest moving the link so it's part of this paragraph.

```suggestion
For additional information, consult:
```